### DISCUSSION TITLE
A letter to open-source maintainers

### DISCUSSION THREAD

**Discussion by giscus** [Xuanwo/blog#436] [Announcements]
# A letter to open-source maintainers

An infrastructure engineer, focused on distributed storage system

https://xuanwo.io/2024/10-a-letter-to-open-source-maintainers/

**Comment by erwin** [comment]
In the FSF `free software` sense (every corporation should have unlimited rights to USE _your_ `free-software`) the "quit the second your not happy" manta is the only rational pushback, however I don't think that many of the developers of successful projects or even the maintainers that have taken over other successful projects have feelings that really line up with the "just walk away then" sentiment you express.

In my view, all of us developers working to normalize some for of "[fair code](https://faircode.io/)" licensing that isn't **free upgrades forever for big business** is the only real long term solution, and if that means that big business again separates itself from indie developers, then good riddance.

The most able to pay, should pay. Simple as.

**Comment by noirgif** [comment]
Unrelated, but I felt the same when I quit gacha games...
